example-context/example-persistence/src/test/java/uk/gov/justice/services/example/cakeshop/persistence/RecipeRepositoryTest.java

```diff
@@ -112,6 +115,36 @@ public void shouldReturnAllRecipes() {
         assertThat(recipeList, hasItems(recipeC));
     }
 
+    @Test
+    public void removeTest() {
+        List<Recipe> recipeList = recipeRepository.findAll();
+
+        assertThat(recipeList, hasSize(3));
+        assertThat(recipeList, hasItems(recipeA));
+        assertThat(recipeList, hasItems(recipeB));
+        assertThat(recipeList, hasItems(recipeC));
+
+        Recipe recipeAA = recipeRepository.findBy(recipeA.getId());
```

Comment:
we can move this to a private method, which will help avoid the variables names too.